README.md

```diff
@@ -19,7 +19,10 @@ This Kibana visualization plugin is like a Data Table, but with enhanced feature
   - More documentation [here](#computed-column-formula--lines-computed-filter-documentation)
 - Filter table lines based on a computed formula (ex: `col0 > 0`)
   - More documentation [here](#computed-column-formula--lines-computed-filter-documentation)
-- Hide some table columns (ex: `0,1` hides columns 0 and 1)
+- Hide some table columns (ex: `0,1,My_Column` hides columns 0, 1 and the column labeled 'My_Column')
+  - Note that the column label must be written as is (including whitespaces), without any superscript/apostrophe.
+  - Column labels containing commas are not supported since the comma is used to separate the columns
+  - It is strongly recommended to use column labels to hide columns rather than their indeces. Using the Split By Cols feature, in fact, indeces might   depend on the global timerange that has been set. 
```

Comment:
replace 'indeces' by 'index'
replace *Split By Cols feature* by *'Split cols' feature*

Reply:
Actually, I put the plural (with a typo since it should be `indices`) since it sounds better (`labels` is plural too). I can change it if you want but it would sound strange (like saying "it's recommended to use cars rather than bicycles when it rains", you wouldn't say "bicycle").

I'll definitely change `Split By cols` into `'Split cols'`

Reply:
OK for indices.